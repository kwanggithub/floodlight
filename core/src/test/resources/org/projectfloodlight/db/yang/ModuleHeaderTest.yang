module ModuleHeaderTest {
    yang-version 1.0;
    namespace org.projectfloodlight.db;
    prefix "mht";
    organization "www.openflowhub.org";
    contact "Paddy Huntington";
    description "Test module headers";
    reference "Test reference";
}