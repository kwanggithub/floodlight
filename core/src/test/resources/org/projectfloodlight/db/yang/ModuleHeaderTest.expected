{
  "name" : "ModuleHeaderTest",
  "yangVersion" : "1.0",
  "organization" : "www.openflowhub.org",
  "contact" : "Paddy Huntington",
  "description" : "Test module headers",
  "reference" : "Test reference",
  "namespace" : "org.projectfloodlight.db",
  "prefix" : "mht"
}